milli/src/index.rs

```diff
@@ -1562,46 +1553,32 @@ impl Index {
         self.main.remap_key_type::<Str>().delete(txn, main_key::PROXIMITY_PRECISION)
     }
 
-    /* script  language docids */
-    /// Retrieve all the documents ids that correspond with (Script, Language) key, `None` if it is any.
-    pub fn script_language_documents_ids(
+    pub fn localized_attributes_rules(
         &self,
         rtxn: &RoTxn<'_>,
-        key: &(Script, Language),
-    ) -> heed::Result<Option<RoaringBitmap>> {
-        self.script_language_docids.get(rtxn, key)
+    ) -> heed::Result<Option<Vec<LocalizedAttributesRule>>> {
+        self.main
+            .remap_types::<Str, SerdeBincode<Vec<LocalizedAttributesRule>>>()
```

Comment:
is using Bincode necessary here? Bincode is a format that isn't reflexive, so if we ever need to modify the format of what is being stored, it might be complicated to do manually.

serde_json is going to be longer, but I don't think it is necessarily an issue here?